charts/x509-certificate-exporter/templates/psp.yaml

```
@@ -38,6 +40,8 @@ apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
 metadata:
   name: {{ printf "%s-%s" (include "x509-certificate-exporter.fullname" $) $dsName }}
+  labels:
+    {{- include "x509-certificate-exporter.labels" . | nindent 4 }}
```

It's in a loop, so this should be
```suggestion
    {{- include "x509-certificate-exporter.labels" $ | nindent 4 }}
```

instead or template inclusion will fail.